spec/models/ip_spec.rb

```diff
@@ -14,14 +14,14 @@
         it "does not save when address is an invalid IP" do
           expect(Ip.count).to eq(0)
           expect(ip.errors.full_messages).to eq([
-            "Address must be a valid IPv4 address (without subnet)"
+            "Address 'invalidIP' is not valid"
```

Comment:
What do the single quotes do in this context. Is this another form of string interpolation?